scripts/inline_comments_data_collection.py

```diff
@@ -206,8 +177,15 @@ def main():
     os.makedirs("patches", exist_ok=True)
     os.makedirs("data", exist_ok=True)
 
+    diff_id_to_revisions_map, diff_phid_to_id = load_revisions_maps()
+
     with open(phabricator.FIXED_COMMENTS_DB, "wb") as dataset_file_handle:
-        for data in process_comments(limit=limit, diff_length_limit=diff_length_limit):
+        for data in process_comments(
+            limit=limit,
+            diff_length_limit=diff_length_limit,
+            diff_id_to_revisions_map=diff_id_to_revisions_map,
+            diff_phid_to_id=diff_phid_to_id,
```

Comment:
nit: given the maps are only used in `process_comments`, you could create them in that function

Reply:
Done here: 931333c920ee853a98be1e0a2f7bd9d193a31d37